### PR TITLE
Draft bionic document

### PR DESCRIPTION
**Pull request by zhengphou**

### CODE REVIEW

File: user/reference/bionic.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+### Third party apt-repositories removed
+
+While third party apt-repositories are used during the image provisioning, they are all removed from the Bionic build image. This has two benefits; a) reduced risk of unrelated interference and b) faster apt-get updates.
+
+To specify a third party apt-repository, you can [add the source with the apt addon](/user/installing-dependencies/#adding-apt-sources) and specify the packages. For example:
+
+```yaml
+dist: bionic
+group: dev
```
Comment (MariadeAnton) [inline user/reference/bionic.md]: @zhengpenghou when is the `group: dev` tag needed?

File: user/reference/bionic.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+---
+title: The Ubuntu 18.04 Build Environment
+layout: en
+---
+
+## What This Guide Covers
+
+This guide provides an overview of the packages, tools and settings available in the Bionic environment.
+
+## Using Bionic
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
## Using Ubuntu 18.04 (Bionic Beaver)
```

File: user/reference/bionic.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+To route your builds to Ubuntu 18.04 LTS, add the following to your `.travis.yml`:
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
To route your builds to Ubuntu 18.04 LTS, Bionic, add the following to your `.travis.yml`:
```

File: user/reference/bionic.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+```yaml
+dist: bionic
+```
+{: data-file=".travis.yml"}
+
+Please note that Bionic is available on our hosted fully virtualized
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
> Please note that Bionic is available at travis-ci.com and travis-ci.org, on the fully virtualized
```

File: user/reference/bionic.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+infrastructure. If you are running an Enterprise installation, please reach out
+to [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com?subject=Try%20out%20Bionic) to see how you can use the Bionic Docker images.
```
Comment (MariadeAnton) [inline user/reference/bionic.md]: @JaroslawSokol is Bionic on enterprise already available? Shall we have dedicated public instructions for it? Thank you!

File: user/reference/bionic.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+## Differences from the previous relealse images
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
## Differences from the previous release images
```

File: user/reference/bionic.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+Travis CI Ubuntu 18.04 includes the following changes and improvements:
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
Travis CI Ubuntu 18.04, Bionic, includes the following changes and improvements:
```

File: user/reference/bionic.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+| git-ppa              | `ppa:git-core/ppa`           |
+| haskell              | `ppa:hvr/ghc`                |
+| mongodb              | `mongodb-4.0-binoic`         |
+| pollinate            | `ppa:pollinate/ppa`          |
+| redis                | `ppa:chris-lea/redis-server` |
+{: style="width: 80%" }
+
+### Services disabled by default
+
+On our Ubuntu 18.04 based infrastructure, to speed up boot time and improve performance we've disabled all services, including the ones that are started by default.
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
On the Ubuntu 18.04 based environment, to speed up boot time and improve performance we've disabled all services by default.
```

File: user/reference/bionic.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+Add any services that you want to start by default to your `.travis.yml`:
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
Add any services that you want to start to your `.travis.yml`:
```

File: user/reference/bionic.md [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+```yaml
+services:
+  - mysql
+  - redis
+```
+{: data-file=".travis.yml"}
+
+## Environment common to all Ubuntu 18.04 images
+
+The following versions of Docker, version control software and compilers are present on all builds, along with more language specific software described in more detail in each language section.
```
Comment (MariadeAnton) [inline user/reference/bionic.md]:
```suggestion
The following versions of Docker, version control software and compilers are present on all Ubuntu 18.04 builds, along with more language specific software described in more detail in each language section.
```

File: user/reference/bionic.md (hunk 1 above)
Comment (MariadeAnton) [reply user/reference/bionic.md]: @zhengphou @zhengpenghou ^^

Comment (zhengphou) [reply user/reference/bionic.md]: Yes. it's needed for now as a preview release.

Comment (plaindocs) [reply user/reference/bionic.md]: So then we need to add it everywhere we use `dist: bionic`, such as in the Snap pages I think? And we should probably explain on this page why it is there.

Comment (zhengphou) [reply user/reference/bionic.md]: If plan to use Ubuntu 18.04,  bionic is the one should be used to choose the right image.  This information has been updated in relevant pages.

Comment (plaindocs) [reply user/reference/bionic.md]: Right, but I'm still talking about `group: dev`.

Comment (zhengphou) [reply user/reference/bionic.md]: we have 3 group tags now, stable, edge and dev. @MariadeAnton your thoughts?

Comment (plaindocs) [reply user/reference/bionic.md]: Maria is away for another week. 
@zhengphou do you want to add a tiny note to this document detail when to use `dev` and see how we go from there?

File: user/reference/bionic.md (hunk 5 above)
Comment (JaroslawSokol) [reply user/reference/bionic.md]: No, there is no support for Bionic available in last Enterprise release (2.2.11).
I think we should not mention it yet for Enterprise.
cc @danishkhan

File: user/reference/bionic.md (hunk 1 above)
Comment (zhengphou) [reply user/reference/bionic.md]: Done

File: user/reference/bionic.md (hunk 5 above)
Comment (zhengphou) [reply user/reference/bionic.md]: The note being removed

File: user/reference/bionic.md [hunk 11]
```diff
@@ -13,12 +13,11 @@ To route your builds to Ubuntu 18.04 LTS, Bionic, add the following to your `.tr
...
 ```yaml
 dist: bionic
+group: dev
 ```
 {: data-file=".travis.yml"}
 
-> Please note that Bionic is available at travis-ci.com and travis-ci.org, on the fully virtualized
-infrastructure. If you are running an Enterprise installation, please reach out
-to [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com?subject=Try%20out%20Bionic) to see how you can use the Bionic Docker images.
+> Please note that Bionic support is newly being built, to use it, **dev** is needed.
```
Comment (Pezi777) [inline user/reference/bionic.md]: Unfortunately, I don't understand what "Bionic support is newly being built" means?

**Pull request merged**